CHANGELOG_iOS.md

```diff
@@ -19,4 +19,21 @@
  - Updated Webcompat report to use the site URL instead of using internal://local. ([#8667](https://github.com/brave/brave-ios/issues/8667))
  - Fixed an issue with muting/unmuting when Brave Talk is minimized. ([#8717](https://github.com/brave/brave-ios/issues/8717))
  - Fixed an issue with the search suggestions button not updating when pressed in the search field. ([#8630](https://github.com/brave/brave-ios/issues/8630))
- 
+
+## [1.64.123](https://github.com/brave/brave-browser/releases/tag/v1.64.123)
+
+### Web3
+
+ - Implemented caching balances for assets in “Portfolio” page. ([#35986](https://github.com/brave/brave-browser/issues/35986))
+ - Removed decimal values for NFTs on “Send” screen. ([#36771](https://github.com/brave/brave-browser/issues/36771))
+ - Removed “Swap” option from all Filecoin Accounts. ([#37167](https://github.com/brave/brave-browser/issues/37167))
+
+### General
+
+ - Added support for Catalan language. ([#36261](https://github.com/brave/brave-browser/issues/36261))
+ - Upgraded Chromium to 123.0.6312.58. ([#36938](https://github.com/brave/brave-browser/issues/36938)) ([Changelog for 123.0.6312.58](https://chromium.googlesource.com/chromium/src/+log/122.0.6261.128..123.0.6312.58?pretty=fuller&n=1000))
```

Comment:
Chromium upgrades should always be at the bottom similar to https://github.com/brave/brave-browser/blob/master/CHANGELOG_DESKTOP.md & https://github.com/brave/brave-browser/blob/master/CHANGELOG_ANDROID.md.

Reply:
Oh i see. Thanks.